src/main/kotlin/frc/robot/subsystems/drive/Drive.java

```diff
@@ -343,23 +420,53 @@ public double getFFCharacterizationVelocity() {
         return output;
     }
 
-    /** Returns the current odometry pose. */
+    /**
+     * Returns the current odometry pose.
+     */
     @AutoLogOutput(key = "Odometry/Robot")
     public Pose2d getPose() {
         return poseEstimator.getEstimatedPosition();
     }
 
-    /** Returns the current odometry rotation. */
+    /**
+     * Returns the current odometry rotation.
+     */
     public Rotation2d getRotation() {
         return getPose().getRotation();
     }
 
-    /** Resets the current odometry pose. */
+    public Command updateDesiredHeading(DoubleSupplier omegaAxis) {
+        return Commands.run(
+                () -> {
+                    double desiredDeltaOmega =
+                            MathUtil.applyDeadband(omegaAxis.getAsDouble(), 0.15)
+                                    * TunerConstants.kMaxOmegaVelocity.in(RadiansPerSecond)
+                                    * ConstantsKt.LOOP_TIME;
+                    desiredHeading = desiredHeading.plus(Rotation2d.fromRadians(desiredDeltaOmega));
```

Comment:
```suggestion
                    desiredHeading = desiredHeading + Rotation2d.fromRadians(desiredDeltaOmega);
```
Also, wouldn't it be simpler to use a measure here?

Reply:
A measure here will just make it longer and more unreadable.

Reply:
Why did you resolve the conversation?

Reply:
Ok now i realized that's not your code (and it's not even in Kotlin lol).